geode-management/src/main/java/org/apache/geode/management/api/ClusterManagementOperation.java

```diff
@@ -29,12 +31,15 @@
  * @param <V> the result type of the operation
  */
 @SuppressWarnings("unused")
-public interface ClusterManagementOperation<V extends OperationResult> {
+public interface ClusterManagementOperation<V extends OperationResult>
```

Comment:
I know this isn't introduced by this PR, but now I am wondering: V really isn't used by this interface, why do we need to declare it here?

Reply:
> I know this isn't introduced by this PR, but now I am wondering: V really isn't used by this interface, why do we need to declare it here?

We need this to maintain the coupling between the operation and the result in the implementations of `ClusterManagementOperation`.